controller/src/main/java/io/pravega/controller/server/security/auth/StreamAuthParams.java

```diff
@@ -76,7 +76,7 @@ public StreamAuthParams(@NonNull String scope, @NonNull String stream, @NonNull
                 return AuthHandler.Permissions.READ;
             } else {
                 if (isMarkStream()) {
-                    return this.requestedPermission(AuthHandler.Permissions.READ_UPDATE);
+                    return AuthHandler.Permissions.READ;
```

Comment:
we can remove method ` requestedPermission(AuthHandler.Permissions defaultValue)`

Reply:
Done.